frontend/src/components/ProjectEditor.vue

```diff
@@ -275,15 +315,15 @@ const sendUnselectedFrames = async () => {
 
 
     const includedFrames: FrameToInclude[] = allFrames.value
-      .filter(frame => !selectedFrames.value.some(selectedFrame => selectedFrame.frameNumber === frame.frameNumber))
-      .map(frame => ({ frame_number: frame.frameNumber, frame_weight: frame.weight }));
+        .filter(frame => !selectedFrames.value.some(selectedFrame => selectedFrame.frameNumber === frame.frameNumber))
+        .map(frame => ({frame_number: frame.frameNumber, frame_weight: frame.weight}));
     console.log('Frame Numbers to Include:', includedFrames);
 
     const payload: CreateLongExposureImageRequest = {
       video_id: uploadedVideoUUID.value,
       frames_to_include: includedFrames,
     };
-    const response: AxiosResponse<string> = await axios.post<CreateLongExposureImageRequest,AxiosResponse<string>>(`${BACKEND_URL}/sendFrames`, payload);
+    const response: AxiosResponse<string> = await axios.post<CreateLongExposureImageRequest, AxiosResponse<string>>(`${BACKEND_URL}/createLongExposureImage`, payload);
```

Comment:
Ah gut, das ist natürlich ein Argument für Axios das es Generics kann und somit ordentlich getypt werden kann...
Wobei ich den Type eher für die response interessant finde und nicht für die Request. Die wird ja in dem Moment eigentlich abgegeben und nicht mehr gebraucht.

Man kann natürlich auch mit `fetch` typen, aber eben nicht über Generics